hippod/api_object_get_detail.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# coding: utf-8
+
+import json
+import os
+import datetime
+import inspect
+import time
+import zlib
+import sys
+
+import aiohttp
+import asyncio
+
+import hippod.hasher
+import hippod.ex3000
+import hippod.api_shared
+
+from hippod.error_object import *
+
+
+def object_index_read(app):
```

Comment:
object_index_read() and other shared functions should be putted somewhere else and shared (api_object_get()) will use the identical functionality. E.g. object_file_ops.py